src/docs/design/intro.mdx

```diff
@@ -4,6 +4,10 @@ import { Divider } from '../../app/components';
 
 Using the PX Blue design system as a designer can greatly improve your efficiency and the quality of your products. PX Blue has many patterns and guidelines (color usage, typography, iconography, etc.) to address the most common elements and interactions that you might encounter when designing a product. By following these recommendations and using pre-defined resources from PX Blue, your time is free to focus on the elements that make your project unique. In general, we recommend trying to use PX Blue for 80% of your product's needs.
 
+## Environment
+
+We currently actively host and maintain our design assets via Figma. Figma is a powerful prototyping tool that facilitates team collaborations and design assets sharing. It works on both browsers and, if you prefer, a desktop app (Mac and PC). You may [sign up here](https://www.figma.com/).
```

Comment:
```suggestion
We host and maintain our design assets through Figma. Figma is a powerful prototyping tool that facilitates collaboration and sharing of design assets between teams. It can be used on the web, and they also provide a desktop app for Mac and Windows. In order to use Figma, you will need to [sign up](https://www.figma.com/) for an account.
```